### DISCUSSION TITLE
suggest add htm/pdf documents for every versions download

### DISCUSSION THREAD

**Discussion by l1t1** [duckdb/duckdb#7612] [General]
like postgresql did

**Comment by Mause** [comment]
Do you mean PDFs of the documentation?

**Reply by l1t1** [reply]
yes, or htmls.zip
see https://www.postgresql.org/docs/

**Reply by Mause** [reply]
Can you please raise an issue in the docs repo instead? https://github.com/duckdb/duckdb-web/
